sections/back_cover.tex

```diff
@@ -5,3 +5,9 @@
     \includegraphics[height=\paperheight, keepaspectratio]{\layout/back.jpg}
   };
 \end{tikzpicture}
+
+\begin{figure*}[!hb]
+  \hypertarget{Difficulty Table}{}
+  \centering
+  \includegraphics[width=\linewidth]{\tables/field_difficulty.png}
+\end{figure*}
```

Comment:
```suggestion
\begin{figure}[!h]
  \hypertarget{Difficulty Table}{}
  \centering
  \includegraphics[width=\linewidth]{\tables/field_difficulty.png}
\end{figure}
```
I think most of those configs are unnecessary. With this only it should work, and it won't confuse `po4a`?